index.html

```diff
@@ -6,216 +6,266 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
     <title>My Portfolio</title>
 </head>
 
 <body>
     <header>
         <nav>
             <span class="logo">Welcome</span>
-            <img src="./img/Union.svg" alt="Hamburger menu icon" />
+            <img class="burger-menu"src="./img/Union.svg" alt="Hamburger menu icon" />
         </nav>
     </header>
 
+    <nav class="head">
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+
     <section class="headline">
         <div class="headline-content">
-            <h1 class="headline-title">Hey there. I'm Patrick<br>
+            <h1 class="headline-title">
+                Hey there. I'm Patrick<br />
                 <span style="color: #36b37e">I'm a software developer</span>
             </h1>
-            <p>
-                I can help you build a product , feature or website Look through some of
-                my work and experience! If you like what you see and have a project you
-                need coded, don't hestiate to contact me.
+            <p class="change">
+                I can help you build a product , feature or website Look through some
+                of my work and  experience! If you like what you see and have a project
+                you need coded, don't hestiate to contact me.
             </p>
```

Comment:
- [ ] Please make sure that your mobile version stay the same. but I think there is some thing that to be improved in your mobile version design.
### Your design👇
![toolbarst](https://user-images.githubusercontent.com/66992901/201480872-451188da-8791-450f-a8d4-fa1437a8216c.PNG)
### figma template👇
![toolbarfigma](https://user-images.githubusercontent.com/66992901/201480894-07d39107-7e74-411a-8d78-e065397081b9.PNG)